src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedString = '';
+    const divide = separator;
```

Comment:
There's no need to use this constant at all 🙂

Reply:
agree, I don't see where you change this variable or some reason, why you can use one more variable for storing separator
so this variable is redundant, delete it